modules/ballerina-lang/src/main/java/org/ballerinalang/model/tree/AnnotationAttachmentNode.java

```diff
@@ -18,7 +18,7 @@
 package org.ballerinalang.model.tree;
 
 import org.ballerinalang.model.tree.expressions.AnnotationAttachmentAttributeNode;
-import org.ballerinalang.model.tree.expressions.AnnotationAttachmentAttributeValueNode;
+import org.wso2.ballerinalang.compiler.tree.expressions.BLangAnnotAttachmentAttribute;
```

Comment:
Shall we not use the implementation class (org.wso2.* packages) here, since this is the public interface?

Reply:
thanks supun, I'll fix it.